OpenRA.Game/Traits/World/ActorMap.cs

```diff
@@ -94,25 +94,31 @@ class ProximityTrigger : IDisposable
 
 			readonly Action<Actor> onActorEntered;
 			readonly Action<Actor> onActorExited;
+			readonly bool considerVRange;
 
 			WPos position;
 			WDist range;
+			WDist vRange;
```

Comment:
Note: this supersedes my previous suggestion of `-1` (although that would also be fine)